Chore: Remove components from the graveyard folder in grafana/ui

ryantxu opened the pull request:
There are a few deprecated components in grafana/ui that are really internal bits used by the angular graph panel and old versions of the uplot timesers panel.  This PR removes them from the `grafana/ui` bundle in grafana 11


# Release notice breaking change

The internal/deprecated components for `GraphNG`, and `GraphContextMenu` have been removed.  The recommended approach to include panels in a component is via PanelRenderer

ryantxu committed:
remove graveyard

ryantxu committed:
Chore: Remove the deprecated Vector type (#83469)

leeoniya commented:
![kill-it-with-fire-fire](https://github.com/grafana/grafana/assets/43234/a29784df-6f13-47bd-bc49-a0170c5c9cd2)

ryantxu committed:
put back GraphNG

ryantxu committed:
put back GraphNG

svennergr committed:
Loki: Fix failing test waiting for `Loading...` copy (#83547)

fix monaco tests

baldm0mma committed:
remove-ui-graveyard-components/ update codeowners

scottlepp committed:
Expressions: Sql expressions with Duckdb (#81666)

duckdb temp storage of dataframes using parquet and querying from sql expressions
---------

Co-authored-by: Ryan McKinley <ryantxu@gmail.com>

srperf committed:
Docs: Add missing visualizations to Grafana vizualization index page (#83351)

Co-authored-by: Nathan Marrs <nathanielmarrs@gmail.com>
Co-authored-by: Isabel Matwawana <76437239+imatwawana@users.noreply.github.com>
Co-authored-by: jev forsberg <jev.forsberg@grafana.com>

ryantxu committed:
remove more of it

ryantxu committed:
Merge remote-tracking branch 'origin/main' into remove-ui-graveyard-components

ryantxu committed:
remove more of it

ryantxu committed:
remove more of it

mckn reviewed (approved):
LGTM!

Do we have any replacement components that we can reference to? Or these components are most likely not used outside of Grafana?

ryantxu commented:
> Do we have any replacement components that we can reference to?

Not really -- these were pretty weird internal components that sadly have names that encourage usage :(

GraphNG is a graph replacement, but at the layer *after* we have fixed any of the configuration -- that means it breaks anytime options get tweaked
